frontend/src/js/controllers/contactUsCtrl.js

```diff
@@ -0,0 +1,108 @@
+// Invoking IIFE for contact us
+
+(function() {
+
+    'use strict';
+
+    angular
+        .module('evalai')
+        .controller('contactUsCtrl', contactUsCtrl);
+
+    contactUsCtrl.$inject = ['utilities', '$state', '$http', '$rootScope'];
+
+    function contactUsCtrl(utilities, $state, $http, $rootScope) {
+        var vm = this;
+        var userKey = utilities.getData('userKey');
+        vm.wrnMsg = {};
+        vm.isValid = {};
+        vm.user = {};
+        vm.isFormError = false;
+
+        vm.startLoader = function(msg) {
```

Comment:
@RishabhJain2018 now we have a service for loader so use that service please. Don't use rootscope here.